mne/epochs.py

```diff
@@ -2002,6 +2008,7 @@ def __init__(self, raw, events, event_id=None, tmin=-0.2, tmax=0.5,
         # proj is on when applied in Raw
         proj = proj or raw.proj
 
+        self._segment_reject = segment_reject
```

Comment:
No need to make it private.

self. reject_by_annotation = reject_by_annotation